src/Aquifer.API/Endpoints/Bibles/Versification/Request.cs

```diff
@@ -0,0 +1,11 @@
+namespace Aquifer.API.Endpoints.Bibles.Versification;
+
+public class Request
+{
+    public int TargetBibleId { get; set; }
```

Comment:
Note for future work: We may want `SourceBibleId` in order to do arbitrary mappings but omitting is fine for now.